grpc/CMakeLists.txt

```diff
@@ -23,10 +24,17 @@ ExternalProject_Add(
   GIT_CONFIG submodule.recurse=1 submodule.fetchJobs=10
   GIT_SHALLOW 1
   SOURCE_DIR grpc_build
-  CONFIGURE_COMMAND ""
+  CMAKE_ARGS
+  -DBUILD_TESTING=OFF
```

Comment:
Could you check the options to enable shared libraries such as `BUILD_SHARED_LIBS`?
By enabling those options, I hope the output libraries are going to be the same as `make`.

Reply:
Sure. Enabled `BUILD_SHARED_LIBS` options and checked it worked.

Reply:
Now the installed libraries look like the attached picture:

![image](https://github.com/user-attachments/assets/c1730d4a-cb70-4957-886f-2dad8d3b9cfb)